_authors/vvavdiya.markdown

```diff
@@ -0,0 +1,11 @@
+---
+- **name:** Vijay Vavdiya
+- **short_name:** Vavdiya
+- **photo:** "/assets/media/authors/vvavdiya.jpeg"
+- **github:** "vvavdiya"
+- **linkedin:** "vijayvavdiya"
```

Comment:
Remove the `**` and `""` from this section - this is front matter that will be parsed as YAML.